### PR TITLE
Adding ONS NA 2019 event.

### PR DESCRIPTION
**Pull request by nickolaev**
Signed-off-by: Nikolay Nikolaev <nnikolay@vmware.com>

### CODE REVIEW

File: content/events/onsna2019.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+-----------------
+* 11:50am - 12:20pm - [Panel Discussion: Towards Kubernetes as the Network Service Orchestrator] (https://sched.co/LKUa) - Heather Kirksey, Ramki Krishnan, Ed Warnicke, Jeffrey Saelens, Nikolay Nikolaev
+
+# Happy NSM Hour
+
+* TBD
+
+# Supporting Collatoral
+* Slides
+  * [Network Service Mesh Intro](https://docs.google.com/presentation/d/1Vzmhv5vc10NyAa08ny-CCbveo0_fWkDckbkCD_N0fPg/edit)
```
Comment (edwarnicke) [inline content/events/onsna2019.md]: We will probably update the slides... so not *quite* these slides :)

Comment (nickolaev) [reply content/events/onsna2019.md]: Done.

**Comment by edwarnicke** [comment]
The preview is missing the image: https://deploy-preview-78--agitated-bose-995577.netlify.com/events/

File: content/events/onsna2019.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
++++
+title = "Network Service Mesh at Open Networking Summit NA 2019"
+link = "https://onsna19.sched.com"
+dates = "April 3-5, 2019"
+image = "/img/events/logo_ons_na.png"
+summary = "Network Service Mesh is joining Open Netowrking Summit in San Jose, California with project introduction and a panel session."
```
Comment (edwarnicke) [inline content/events/onsna2019.md]: Open Networking Summit, not Open Netowrking Summit.

**Comment by nickolaev** [comment]
> The preview is missing the image: https://deploy-preview-78--agitated-bose-995577.netlify.com/events/

Not sure how the preview works here. The image is shown on the local `hugo server`.

**Comment by edwarnicke** [comment]
Also works locally for me.  Merging.

**Pull request merged**